pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/stats/MetadataStoreStats.java

```diff
@@ -24,48 +24,32 @@
 
 public final class MetadataStoreStats implements AutoCloseable {
     private static final double[] BUCKETS = new double[]{1, 3, 5, 10, 20, 50, 100, 200, 500, 1000, 2000, 5000};
-    private static final String LABEL_NAME = "name";
+    private static final String OPS_TYPE_LABEL_NAME = "type";
+    private static final String METADATA_STORE_LABEL_NAME = "name";
+    private static final String OPS_TYPE_GET = "get";
+    private static final String OPS_TYPE_DEL = "del";
+    private static final String OPS_TYPE_PUT = "put";
     protected static final String PREFIX = "pulsar_metadata_store_";
 
-    private static final Histogram GET_OPS_SUCCEED = Histogram
-            .build(PREFIX + "get_latency", "-")
+    private static final Histogram OPS_SUCCEED = Histogram
+            .build(PREFIX + "ops_latency", "-")
             .unit("ms")
             .buckets(BUCKETS)
-            .labelNames(LABEL_NAME)
+            .labelNames(METADATA_STORE_LABEL_NAME, OPS_TYPE_LABEL_NAME)
```

Comment:
Can you explain the meaning of the metadata store name? I haven't seen in the implementation of metadata stores any name.

Reply:
there will be more than 1 Metadata store, one for broker metadata, one for dynamic config.